lib/json_schema/configuration.rb

```diff
@@ -1,15 +1,26 @@
 module JsonSchema
   class Configuration
     attr_reader :validate_regex_with
+    attr_reader :custom_formats
```

Comment:
Sorry for the nitpick, but mind just keeping the `attr_reader` list and method names in this file alphabetized?